packages/shared/lib/services/user.service.ts

```diff
@@ -136,6 +137,18 @@ class UserService {
     async getInvitedUserByToken(token: string): Promise<InviteUser | null> {
         const date = new Date();
 
+        if (isEnterprise() && process.env['NANGO_ADMIN_INVITE_TOKEN'] === token) {
+            return {
```

Comment:
can we return null instead of a empty user?

Reply:
Well, if we return null then the path would think there is no actual user so it is an empty user to get around that.

Reply:
I think once again, a good env var (or a computed env var) would be easier to follow. Ultimately those are all feature flags that could be centralized.